test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSViewAccessibility.cs

```diff
@@ -228,72 +93,6 @@ public void ViewAccessibilityAccessibilityRangeContent()
             tlog.Debug(tag, $"ViewAccessibilityAccessibilityRangeContent END (OK)");
         }
 
-        [Test]
-        [Category("P1")]
-        [Description("ViewAccessibility.View.AccessibilityAttributes.Remove.")]
-        [Property("SPEC", "Tizen.NUI.ViewAccessibility.View.AccessibilityAttributes.Remove M")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "MR")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void ViewAccessibilityViewRemoveAccessibilityAttribute()
-        {
-            tlog.Debug(tag, $"ViewAccessibilityViewRemoveAccessibilityAttribute START");
-
-            var testingTarget = new View();
-            Assert.IsNotNull(testingTarget, "Can't create success object View");
-            Assert.IsInstanceOf<View>(testingTarget, "Should be an instance of View type.");
-
-            testingTarget.AccessibilityAttributes["MyView"] = "test";
-
-            try
-            {
-                testingTarget.AccessibilityAttributes.Remove("MyView");
-            }
-            catch (Exception e)
-            {
-                tlog.Debug(tag, e.Message.ToString());
-                Assert.Fail("Caught Exception: Failed!");
-            }
-
-            Assert.IsEmpty(testingTarget.AccessibilityAttributes);
-
-            testingTarget.Dispose();
-            tlog.Debug(tag, $"ViewAccessibilityViewRemoveAccessibilityAttribute END (OK)");
-        }
-
-        [Test]
-        [Category("P1")]
-        [Description("ViewAccessibility.View.AccessibilityAttributes.Clear.")]
-        [Property("SPEC", "Tizen.NUI.ViewAccessibility.View.AccessibilityAttributes.Clear MR")]
-        [Property("SPEC_URL", "-")]
-        [Property("CRITERIA", "MR")]
-        [Property("AUTHOR", "guowei.wang@samsung.com")]
-        public void ViewAccessibilityViewClearAccessibilityAttributes()
```

Comment:
This test has just been updated: https://github.com/Samsung/TizenFX/pull/4404

Reply:
> This test has just been updated: #4404

AccessibilityAttributes is protected,  we cannot call it directly, so I rewrite the relevant case, please check.

Reply:
OK